compiler/rustc_feature/src/accepted.rs

```diff
@@ -186,6 +186,8 @@ declare_features! (
     /// Allows some increased flexibility in the name resolution rules,
     /// especially around globs and shadowing (RFC 1560).
     (accepted, item_like_imports, "1.15.0", Some(35120), None),
+    /// Allows `'a: { break 'a; }`.
+    (accepted, label_break_value, "1.64.0", Some(48594), None),
```

Comment:
Nightly is already 1.65 now.

Reply:
#100591 can't come fast enough :laughing: